packages/libs/eda/src/lib/core/components/visualizations/implementations/VolcanoPlotVisualization.tsx

```diff
@@ -448,6 +483,64 @@ function VolcanoPlotViz(props: VisualizationProps<Options>) {
     ...(data.value ? {} : EMPTY_VIZ_AXIS_RANGES),
   };
 
+  // set truncation flags: will see if this is reusable with other application
+  const {
+    truncationConfigIndependentAxisMin,
+    truncationConfigIndependentAxisMax,
+    truncationConfigDependentAxisMin,
+    truncationConfigDependentAxisMax,
+  } = useMemo(
+    () =>
+      truncationConfig(
+        {
+          independentAxisRange: defaultIndependentAxisRange,
```

Comment:
Instead of making new default...AxisRange variables, can we just have this memo dependent on the `rawDataMinMaxValues`? The independent axis range would just be 
```
{
    min: rawDataMinMaxValues.x.min,
    max: rawDataMinMaxValues.x.max,
}
```
like you have above, and then the dependent axis range could be
```
{
    min: -Math.log10(rawDataMinMaxValues.y.max),
    max: -Math.log10(rawDataMinMaxValues.y.min)
}
```
.


I'm hoping that will keep the component simpler, since we already have a few variables that have to do with the range of the data or axes, and i have trouble keeping them all straight as it is :D 
What do you think?